src/ContinuousEnergyNeutron.tex

```diff
@@ -373,10 +386,14 @@ \subsubsection{\textsf{LDLW} \& \textsf{LDLWP} Blocks}\label{sec:LDLWBlock}\labe
 \end{tablenotes}
 \end{ThreePartTable}
 
-\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.} The \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+The corresponding \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+ 
+\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.}
+
+\subsubsection{\textsf{DLW} \& \textsf{DLWP} \& \textsf{DNED} Blocks}\label{sec:DLWBlock}\label{sec:DLWPBlock}\label{sec:DNEDBlock}
```

Comment:
It looks like Tables 21 and 23 need to be updated to include the DNEDL and DNED names (currently, they just say "delayed neutrons")

Reply:
I replaced the mention of delayed neutrons with the actual name of the block